Quick start updates (#19960)
pingcap/docs: quick-start-with-tidb.md

## Deploy a local test cluster

> Scenario: Quickly deploy a local TiDB cluster for testing using a single macOS or Linux server. By deploying such a cluster, you can learn the basic architecture of TiDB and the operation of its components, such as TiDB, TiKV, PD, and the monitoring components.

Reviewer: ```suggestion
This section describes how to quickly deploy a local TiDB cluster for testing on a single macOS or Linux server. By deploying such a cluster, you can learn the basic architecture of the TiDB database and the operation of its components, such as TiDB, TiKV, PD, and the monitoring components.
```

        tiup playground v8.5.0 --db 2 --pd 3 --kv 3
        ```

>         To view the all available versions, run `tiup list tidb`.

Reviewer 2: ```suggestion
        To view all available versions, run `tiup list tidb`.
```

    ```log
    Successfully set mirror to https://tiup-mirrors.pingcap.com
    Detected shell: bash
    Shell profile:  /home/user/.bashrc
    /home/user/.bashrc has been modified to add tiup to PATH
    open a new terminal or source /home/user/.bashrc to use it

>     Installed path: /home/user/.tiup/bin/tiup

Reviewer: For reviewer: This is to align the output with that in Chinese docs.

## Simulate production deployment on a single machine

> Scenario: Experience the smallest TiDB cluster with the complete topology and simulate the production deployment steps on a single Linux server.

Reviewer: ```suggestion
This section describes how to set up the smallest TiDB cluster with a full topology, and simulate production deployment steps on a single Linux server.
```

        service sshd restart
        ```

1. Create and start the cluster:

>     Create the [topology configuration file](/tiup-cluster-topology-reference) according to the following template, and name it as `topo.yaml`:

Reviewer 2: ```suggestion
    Create the [topology configuration file](/tiup/tiup-cluster-topology-reference.md) according to the following template, and name it as `topo.yaml`:
```

    Input SSH password:
    ```

1. Start the cluster:

    {{< copyable "shell-regular" >}}

    ```shell
    tiup cluster start <cluster-name>
    ```

1. Access the cluster endpoints:

    - Connect to the TiDB database using MySQL client. The password is empty:

        {{< copyable "shell-regular" >}}

        ```shell
        mysql -h 10.0.1.1 -P 4000 -u root
        ```

    - Grafana: <http://{grafana-ip}:3000>. The default username and password are both `admin`.

    - [TiDB Dashboard](/dashboard/dashboard-intro.md): <http://{pd-ip}:2379/dashboard>. The default username is `root`, and the password is empty.

> 1. To view the currently deployed cluster list:

Reviewer 2: Please use bullet points (`-`) instead of ordered numbers (`1.`, `2.`) here because these items represent independent commands, not sequential steps that must be performed in order:

https://github.com/pingcap/docs/blob/8234ff0ee94654a55bfbdb85fad7b277a09b2bb4/quick-start-with-tidb.md?plain=1#L495-L509
